css/style.css

```diff
@@ -0,0 +1,1386 @@
+  padding: 10px 16px 10px 84px;
+}
+
+.indexCo h3 {
+  margin-top: 10px;
+  width: fit-content;
+  height: fit-content;
+}
+
+.index-img {
+  width: 60px;
+  height: 60px;
+  background-color: blue;
+  display: block;
+  position: relative;
+}
+
+.indexCo li {
+  margin-bottom: 20px;
+  display: block;
+  margin-top: 20px;
+}
+
+.indexCo h4 img {
+  width: 60px;
+  height: 60px;
+  display: block;
+  margin: 10px 10px;
+}
+
+.latest-news p {
+  margin-top: 10px;
+  color: rgb(80, 80, 80);
+  font-size: 12px;
+}
+
+.insights p {
+  color: rgb(255, 255, 255, 0.6);
+  font-size: 12px;
+}
+
+.indexCo h4 p {
+  margin-top: 10px;
+}
+
+.indexCo h6 {
+  font-size: 11px;
+  font-weight: 500;
+  letter-spacing: 0.04em;
+  line-height: 1.5;
+}
+
+.main-a-container {
+  min-width: fit-content;
+}
+
+.index-right {
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+}
+
+.indexCo h4 {
+  background-color: #f42;
+  color: white;
+  font-size: 18px;
+  height: 43px;
+  display: flex;
+}
+
+.img-background {
+  background-image: linear-gradient(to bottom, transparent, black), url("../images/Frankestein.PNG");
+  background-repeat: no-repeat;
+  background-position: bottom;
+  width: 100%;
+  background-size: cover;
+  min-height: 276px;
+}
+
+.author-span {
+  color: rgb(250, 245, 236);
+  text-decoration: none;
+  padding-left: 1em;
+}
+
+.author-span:hover {
+  color: white;
+}
+
+.text-bold {
+  font-size: 20px;
+  font-weight: 600;
+  text-decoration: none;
+  color: white;
+  display: block;
+}
+
+.text-bold:hover {
+  text-decoration: underline;
+}
+
+.text-thumbnail {
+  display: flex;
+  align-items: center;
+  background: rgba(255, 68, 34, 0.08);
+  justify-content: space-between;
+}
+
+.text-thumbnail a {
+  color: #f42;
+  text-decoration: none;
+}
+
+.main-a-list li {
+  margin: 10px auto;
+  border-bottom: 1px solid #aaa;
+  line-height: 1em;
+  padding-bottom: 10px;
+}
+
+.main-a-list li > a {
+  text-decoration: none;
+  color: #555;
+  font-size: 1em;
+}
+
+.thumbnail {
+  min-width: 97px;
+  min-height: 97px;
+  padding-left: 10px;
+}
+
+.main-a-list {
+  list-style: none;
+}
+
+.indexCo h4 a {
+  color: white;
+  display: flex;
+  align-items: center;
+}
+
+.latest-news p a {
+  margin-top: 10px;
+  color: rgb(80, 80, 80);
+  font-size: 12px;
+}
+
+.latest-news h2 a {
+  font-size: 28px;
+  color: #f42;
+}
+
+.indexCo h3 a {
+  font-size: 14px;
+  color: #f42;
+}
+
+.indexCo h2 a {
+  font-size: 28px;
+  color: #f42;
+}
+
+.insights p a {
+  color: rgb(255, 255, 255, 0.6);
+  font-size: 12px;
+}
+
+.insights h5 a {
+  color: white;
+  font-size: 14px;
+  display: block;
+  margin-top: 40px;
+}
+
+.insights h2 a {
+  color: white;
+  font-size: 20px;
+}
+
+.main-a-list li::before {
+  content: "\2022";
+  color: #f42;
+  display: inline-block;
+  width: 1em;
+  margin-left: 0;
+}
+
+/* DESKTOP MEDIA QUERIE */
+
+@media screen and (min-device-width: 767px) and (max-device-width: 2000px) {
```

Comment:
you need to fix this query I have a 4k screen and the website was displaying like this:
![image](https://user-images.githubusercontent.com/34653764/86419675-da980400-bccb-11ea-840f-8d0995c29a64.png)

you can fix it by first change max-device-width and min-device-width by max-width and min-width and give to the max-width a really big value like 20000px so for bigger screens it will work, I wish that you come up with a better solution.